Gemfile

```diff
@@ -20,6 +20,7 @@ group :test do
   gem "govuk-content-schema-test-helpers"
   gem "govuk_schemas"
   gem "minitest-spec-rails"
+  gem "minitest-stub-const"
```

Comment:
Could you add a note to the commit message about why this gem is needed?

Reply:
done! 👍